modules/layers/src/solid-polygon-layer/polygon-tesselator.js

```diff
@@ -42,7 +42,7 @@ export default class PolygonTesselator extends Tesselator {
         positions: {size: 3},
         positions64xyLow: {size: 2, fp64Only: true},
         vertexValid: {type: Uint8ClampedArray, size: 1},
-        indices: {type: IndexType, size: 1}
+        indices: {type: Uint32Array, size: 1}
```

Comment:
Can't recall, is this now automatically handled on lower level (luma.gl attribute?), or are we dropping support for browsers that don't have the extension?